lib/screens/v2/candidate_generator/dup.ex

```diff
@@ -113,86 +134,212 @@ defmodule Screens.V2.CandidateGenerator.Dup do
             secondary_departures: %Departures{sections: secondary_sections}
           }
         } = config,
-        fetch_section_departures_fn
+        now,
+        fetch_section_departures_fn,
+        fetch_alerts_or_empty_list_fn
       ) do
     primary_sections_data =
-      primary_sections
-      |> Task.async_stream(fetch_section_departures_fn, timeout: :infinity)
-      |> Enum.map(fn {:ok, data} -> data end)
+      get_sections_data(
```

Comment:
I think it was a mistake to remove it. It can't stay here because we are no longer working with the enumerable directly. It is now in `get_sections_data`.